ahoy.js

```diff
@@ -359,13 +360,24 @@
   };
 
   ahoy.trackAll = function() {
+    ahoy.start();
```

Comment:
I think you should have to call `ahoy.start()` manually if `startOnReady` is false.